provisioning/terraform/service.tf

```diff
@@ -35,11 +35,16 @@ resource "google_cloud_run_v2_service" "server" {
           path = "/healthy"
         }
       }
+      volume_mounts {
+        name       = "cloudsql"
+        mount_path = "/cloudsql"
+      }
     }
-    annotations = {
```

Comment:
discussion: Looks like we're dropping maxScale and client-name, as opposed to adapting them to a new format.

Is that because these are defaults that don't need to be explicitly set? Might we want to be explicit about maxScale anyway so folks can see that toggle?

Reply:
the maxScale setting was inherited from an older terraform setup where (if memory serves) this value had to be set as otherwise there would be terraform plan deltas on re-applying. 

For client-name, I can't see any examples in the cloud_run_v2_service specs of how to do this, and I'm not sure how much value it adds, all I know is that keeping it in causes errors.

Reply:
Resolving this comment, and opened #257 to follow up

Reply:
I think based on the documentation that for client name it's 
[client](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_v2_service#client) - (Optional) Arbitrary identifier for the API client.

versus
v1 doc [run.googleapis.com/client-name](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_service#run.googleapis.com/client-name) sets the client name calling the Cloud Run API.

It's unclear, and there isn't a sample showing it's use